src/js/themes/base.js

```diff
@@ -315,6 +315,12 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         },
         // deprecate in v3
         // weight: undefined,
+        readOnly: {
```

Comment:
if this isn't applicable to all inputs, would it be better to just have it under the components that are affected?

Reply:
or is it implied that this only will apply to inputs that can support readonly? I could be okay with that too

Reply:
Eventually the idea would be that we would apply this styling to all inputs that support read-only. In this PR though we are just adding support for TextInput and DateInput